File: src/main/java/um/haberes/report/model/haberes/core/LegajoControlDto.java
package um.haberes.report.model.haberes.core;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class LegajoControlDto {

    private Long legajoControlId;

    private Long legajoId;

    private int anho = 0;

    private int mes = 0;

    private byte liquidado = 0;

    private byte fusionado = 0;

    private byte bonoEnviado = 0;

    private PersonaDto persona;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/LegajoControlDto.kt
package um.haberes.report.kotlin.dto.haberes.core

data class LegajoControlDto(

    var legajoControlId: Long? = null,

    var legajoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var liquidado: Byte = 0,
    var fusionado: Byte = 0,
    var bonoEnviado: Byte = 0,
    var persona: PersonaDto? = null

)
